cvat/apps/engine/views.py

```diff
@@ -450,119 +556,109 @@ def _validate_task_limit(self, owner):
-            queue = django_rq.get_queue("default")
-            rq_id = "/api/v1/tasks/{}/export".format(pk)
-
-            rq_job = queue.fetch_job(rq_id)
-            if rq_job:
-                last_task_update_time = timezone.localtime(db_task.updated_date)
-                request_time = rq_job.meta.get('request_time', None)
-                if request_time is None or request_time < last_task_update_time:
-                    rq_job.cancel()
-                    rq_job.delete()
+                # RQ adds a prefix with exception class name
+                import_error_prefix = '{}.{}'.format(
+                    CvatImportError.__module__, CvatImportError.__name__)
+                if exc_info.startswith(import_error_prefix):
+                    exc_info = exc_info.replace(import_error_prefix + ': ', '')
+                    return Response(data=exc_info,
+                        status=status.HTTP_400_BAD_REQUEST)
                 else:
-                    if rq_job.is_finished:
-                        file_path = rq_job.return_value
-                        if action == "download" and osp.exists(file_path):
-                            rq_job.delete()
-
-                            timestamp = datetime.strftime(last_task_update_time,
-                                "%Y_%m_%d_%H_%M_%S")
-                            filename = "task_{}_backup_{}{}".format(
-                                db_task.name, timestamp,
-                                osp.splitext(file_path)[1])
-                            return sendfile(request, file_path, attachment=True,
-                                attachment_filename=filename.lower())
-                        else:
-                            if osp.exists(file_path):
-                                return Response(status=status.HTTP_201_CREATED)
-                    elif rq_job.is_failed:
-                        exc_info = str(rq_job.exc_info)
-                        rq_job.delete()
-                        return Response(exc_info,
-                            status=status.HTTP_500_INTERNAL_SERVER_ERROR)
-                    else:
-                        return Response(status=status.HTTP_202_ACCEPTED)
+                    return Response(data=exc_info,
+                        status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
-            ttl = dm.views.TASK_CACHE_TTL.total_seconds()
-            queue.enqueue_call(
-                func=dm.views.backup_task,
-                args=(pk, 'task_dump.zip'),
-                job_id=rq_id,
-                meta={ 'request_time': timezone.localtime() },
-                result_ttl=ttl, failure_ttl=ttl)
-            return Response(status=status.HTTP_202_ACCEPTED)
+        return Response({'rq_id': rq_id}, status=status.HTTP_202_ACCEPTED)
 
-        else:
+    @action(methods=['GET'], detail=True, url_path='backup')
+    def export(self, request, pk=None):
+        db_task = self.get_object() # force to call check_object_permissions
```

Comment:
@azhavoro , from my perspective import/export for tasks and projects look very similar. Could we unify them? Also I will recommend to create a separate app. Basically here the code can be simple:
```
if request.method == 'GET':
    http_status = Backup.export(...)
else:
    http_status = Backup.import(...)

return Response(status=http_status)
```